### PR TITLE
Getting Started Updates for SIG-Windows

### PR DESCRIPTION
**Pull request by rjmorse**
- Remove the implicit recommendation to compile node binaries
- Clarify the TOR example configuration file
- Updates to sample yaml files and explain the OS tag usage of `1709`
- Add to known issues list
- Remove mentioned required PR that has been merged
- Add next steps footer

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7117)

### CODE REVIEW

**Comment by tengqm** [comment]
@rjmorse pls sign the CLA.

**Comment by rjmorse** [comment]
Thanks @tengqm I'm working to get an org CLA setup

File: docs/getting-started-guides/windows/index.md [hunk 1]
```diff
@@ -16,35 +16,10 @@ The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) co
...
-    KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet
-
-    # Build the kube-proxy
-    KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy
-
-    # You will find the output binaries under the folder _output/local/bin/windows/
-```
-
-More detailed build instructions will be maintained and kept up to date [here](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries.md).
+If you wish to build the code yourself, please refer to detailed build instructions [here](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries.md).
```
Comment (gkudra-msft) [inline docs/getting-started-guides/windows/index.md]: I recommend linking to the [rendered page](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries) instead.

File: docs/getting-started-guides/windows/index.md [hunk 2]
```diff
@@ -111,7 +84,7 @@ To run Windows Server Containers on Kubernetes, you'll need to set up both your
 
 
 1. Windows Server container host running the required Windows Server and Docker versions. Follow the setup instructions outlined by this help topic: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server.
-2. [Build](#Build) or download kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions
+2. [Get Windows Binaries](#Get-Windows-Binaries) kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions
```
Comment (gkudra-msft) [inline docs/getting-started-guides/windows/index.md]: The `#link` needs to be all-lowercase to work.

File: docs/getting-started-guides/windows/index.md [hunk 3]
```diff
@@ -254,6 +227,21 @@ To start your cluster, you'll need to start both the Linux-based Kubernetes cont
...
+
+The kubeadm binary can be found at [Kubernetes Releases](https://github.com/kubernetes/kubernetes/releases), inside the node binaries archive. Adding a Windows node is not any different than adding a Linux node:
+
+`kubeadm.exe join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>`
+
+See [joining-your-nodes](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#44-joining-your-nodes) for more details.
+
+## Supported Features
+
+The examples listed below assume running Windows nodes on Windows Server 1709. If you are running Windows Server 2016, the examples will need the image updated to specify `image: microsoft/windowsservercore`. This is due to the requirement for container images to match the host operating system version when using process isolation.
```
Comment (gkudra-msft) [inline docs/getting-started-guides/windows/index.md]: Who knows when the `:latest` tag won't apply to Server 2016 anymore? `:ltsc2016` is more explicit. Maybe a link to https://hub.docker.com/r/microsoft/windowsservercore/ for further reading would also be useful.

File: docs/getting-started-guides/windows/index.md [hunk 4]
```diff
@@ -439,7 +440,12 @@ Some of these limitations will be addressed by the community in future releases
 - Using Secrets and ConfigMaps as volume mounts is not supported 
 - The StatefulSet functionality for stateful applications is not supported
 - Horizontal Pod Autoscaling for Windows Server Container pods has not been verified to work end-to-end
-- Hyper-V Containers are not supported
+- Hyper-V isolated containers are not supported. 
+- Windows container OS must match the Host OS. If it does not, the pod will get stuck in a crash loop.
+- Under the networking models of L3 or Host GW, Kubernetes Services are inaccessible to Windows nodes due to a Windows issue. This is not an issue if using OVN/OVS for networking.
+- Windows kubelet.exe may fail to start when running on Windows Server under VMWare Fusion [issue 57110](https://github.com/kubernetes/kubernetes/pull/57124)
+- Flannel and Weavenet are not yet supported
```
Comment (gkudra-msft) [inline docs/getting-started-guides/windows/index.md]: Potentially link to the [other FAQ](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems) for more debugging?

File: docs/getting-started-guides/windows/index.md (hunk 1 above)
Comment (rjmorse) [reply docs/getting-started-guides/windows/index.md]: Great point!

File: docs/getting-started-guides/windows/index.md (hunk 2 above)
Comment (rjmorse) [reply docs/getting-started-guides/windows/index.md]: TIL

File: docs/getting-started-guides/windows/index.md (hunk 3 above)
Comment (rjmorse) [reply docs/getting-started-guides/windows/index.md]: This is a much better approach. Implicit tags lead to serious problems

File: docs/getting-started-guides/windows/index.md (hunk 4 above)
Comment (rjmorse) [reply docs/getting-started-guides/windows/index.md]: Added under `## Next steps and resources`

**Comment by PatrickLang** [comment]
LGTM. Thanks!

**Comment by xiangpengzhao** [comment]
@rjmorse please resolve the conflicts and check your CLA status :)

**Comment by michmike** [comment]
/ping @zacharysarah  and @a-mccarthy 
can we get this reviewed please?

**Comment by zacharysarah** [comment]
@rjmorse 👋 Two things:

1. Please replace this PR with a new one and cherry-pick the relevant commits. 

It looks like a previous merge imported a larger commit history. Please reference this PR in the OP so we can shorten the review process. If you `/assign @zacharysarah` in a comment, I'll catch it more quickly. 👍 

2. Please resolve any CLA issues.

It looks like you're authorized as an individual user; are you part of an org with its own CLA permissions?

**Comment by rjmorse** [comment]
Thanks @zacharysarah I'll see if I can get these items squared away today.

I have an individual CLA and I've initiated a org CLA, but since that hasn't been approved it seems like the pending org CLA is blocking me

**Comment by zacharysarah** [comment]
@rjmorse 👋 Any updates?

**Comment by rjmorse** [comment]
I finally created a new PR at https://github.com/kubernetes/website/pull/7535 and the CLA issue, merge conflicts, and improper merge issues have been resolved. PTAL

**Comment by heckj** [comment]
closing this PR since the replacement #7535 was created and merged.

**Pull request closed without merging**